frontend/demo/component/richtexteditor/rich-text-editor-basic.ts

```diff
@@ -22,7 +22,7 @@ export class Example extends LitElement {
       <!-- tag::snippet[] -->
       <vaadin-rich-text-editor
         style="height: 400px;"
-        value=${this.richText}
+        .value=${this.richText}
```

Comment:
Should be fixed for all attribute values.

```suggestion
        .value="${this.richText}"
```

Reply:
Done